lsr_role2collection.py

```diff
@@ -1213,7 +1213,12 @@ def parse_extra_mapping(mapping_str, namespace, collection, role):
                         _dest_name["role"] = _dest[0]
                     elif len(_dest) == 2:
                         # "collection.rolename" or "namespace.collection"
-                        if _dest[0] == namespace and _dest[1] == collection:
+                        if (
+                            _src[0] == "fedora"
+                            and _src[1] == "linux_system_roles"
+                            or _dest[0] == namespace
+                            and _dest[1] == collection
+                        ):
```

Comment:
I think the precedence for `and` is greater than `or`, but please use parentheses for readability:
```
                            (_src[0] == "fedora"
                            and _src[1] == "linux_system_roles")
                            or (_dest[0] == namespace
                            and _dest[1] == collection)
```